theano/tensor/tests/test_subtensor.py

```diff
@@ -97,18 +98,17 @@ def setUp(self):
         Subtensor.debug = False
         utt.seed_rng()
 
-    def eval_output_and_check(self, t, list=False, mode=None):
+    def eval_output_and_check(self, t, op_type=None, mode=None, length=1):
```

Comment:
I don't understand.  Doesn't line 102 just below here set this to self.sub by default?